src/i18n/i18n.ts

```diff
@@ -137,6 +137,7 @@ const dict = {
         refund_explainer: "You will be able to refund after the timeout",
         swap_not_refundable_yet: "Your swap is not refundable yet",
         wasm_not_supported: "Please activate WebAssembly in your browser",
+        ios_image_download_do_not_share: "SECRET! DO NOT SHARE!",
```

Comment:
@kilrau translations are missing for this

Reply:
added